components/content-picker/index.js

```diff
@@ -154,13 +154,15 @@ const ContentPicker = ({
 						{content.length > 1 ? multiPickedLabel : singlePickedLabel}
 					</span>
 
-					<SortableList
-						posts={content}
-						handleItemDelete={onDeleteItem}
-						isOrderable={isOrderable}
-						mode={mode}
-						setPosts={onPickChange}
-					/>
+					<ul className="block-editor-link-control__search-items">
```

Comment:
Right now it was rendering standalone `li` elements. This fixes the semantics here

Reply:
Good catch 👍